itests/src/test/resources/testconfiguration.properties

```diff
@@ -222,6 +222,7 @@ mr.query.files=\
   mapjoin_subquery2.q,\
   mapjoin_test_outer.q,\
   masking_5.q,\
+  msck_repair_filter.q,\
```

Comment:
is there a reason that we run this test with mr?

Reply:
Not a requirement though. This was done before we moved our qtests to run on LLAP by default. It makes sense to move this to LLAP as well.
Fixed.